wpengine/faustjs
# fix: create package entry points to allow import paths without 'dist'

For [BH-775](https://wpengine.atlassian.net/browse/BH-775).

Improves package import paths:

```
import * as api from "@wpengine/headless/api";
import * as auth from "@wpengine/headless/auth";
import * as config from "@wpengine/headless/config";
import { headlessConfig } from "@wpengine/headless/config"; // Named imports also work.
import * as components from "@wpengine/headless/components";
import { Menu } from "@wpengine/headless/components";
import * as provider from "@wpengine/headless/provider";
```

## To test

1. Download and unzip [test-imports.zip](https://github.com/wpengine/headless-framework/files/5711828/test-imports.zip)
2. cd `/path/to/headless-framework/packages/headless && npm link`
3. cd `/path/to/test-imports/ && npm i && git checkout entry-points && npm link @wpengine/headless/`
4. npm run build
5. node `src/index.js`

You should see output from `src/index.ts`.
